RevenueCat/Scripts/Purchases.cs

```diff
@@ -88,7 +101,16 @@ private void Start()
     // Call this if you want to reset with a new user id
```

Comment:
is that comment still valid? wouldn't that be login now? i see this is private anyway but might be confusing

Reply:
I don't think we should tell folks to call setup again! That might be reeeeally outdated, you should be calling `reset` in old SDKs and `logOut` in newer ones, but never setup. Great catch!